app/models/medicaid_application_member_attributes.rb

```diff
@@ -40,6 +40,10 @@ def to_h
         bool_to_checkbox(member.pay_student_loan_interest?),
       :"#{position}_member_deduction_alimony_yes" =>
         bool_to_checkbox(member.pay_child_support_alimony_arrears?),
+      joint_filer_key => "Yes",
```

Comment:
could we add object-level tests for logic about other joint filing member?

Reply:
done at 42daaa993bb04faaa6c973436262c8bee6aa135d